src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -163,15 +163,18 @@ PathObjectPairs joinPredicateAndObject(const VarOrPath& predicate,
 }
 
 // ___________________________________________________________________________
-SparqlExpressionPimpl Visitor::visitExpressionPimpl(auto* ctx,
-                                                    bool allowLanguageFilters) {
+SparqlExpressionPimpl Visitor::visitExpressionPimpl(
+    auto* ctx, [[maybe_unused]] bool allowLanguageFilters) {
   SparqlExpressionPimpl result{visit(ctx), getOriginalInputForContext(ctx)};
+  /*
   if (allowLanguageFilters) {
     checkUnsupportedLangOperationAllowFilters(ctx, result);
   } else {
     checkUnsupportedLangOperation(ctx, result);
   }
   return result;
+  */
```

Comment:
If we don't need this anymore (because we implement filters),
Then you can delete the outcommented code, the unused functions and the unused arguments.